docs/VLSI/Advanced-Usage.rst

```diff
@@ -106,3 +106,18 @@ With the Synopsys plugin, hierarchical RTL and gate-level simulation is supporte
 * ``-$(VLSI_TOP)`` suffixes denote simulations/power analysis on a submodule in a hierarchical flow (remember to override this variable). Note that you must provide the testbenches for these modules since the default testbench only simulates a Chipyard-based ``ChipTop`` DUT instance.
 
 The simulation configuration (e.g. binaries) can be edited for your design. See the ``Makefile`` and refer to Hammer's documentation for how to set up simulation parameters for your design.
+
+UPF Generation Flow
```

Comment:
There should be an intro sentence saying that the VLSI flow experimentally supports generated Chisel-based UPF files using Chisel aspects. (note: all the keywords I put here)(I would also make the UPF and Chisel aspect words hyperlinks to explanations of what they are - or some reference).